.github/workflows/bump-version-command.yml

```
@@ -90,7 +95,8 @@ jobs:
             connectors --modified bump-version \
               ${{ github.event.inputs.type }} \
               "${{ github.event.inputs.changelog }}" \
-              --pr-number ${{ github.event.inputs.pr }}
+              --pr-number ${{ github.event.inputs.pr }} \
+              ${{ github.event.inputs.is-release-candidate == 'true' && '--rc' || '' }}
```

Maybe something like...

```suggestion
              ${{ github.event.inputs.type == 'rc' && '--rc' || '' }}
```